src/VisualStudio/Core/Def/Implementation/ProjectSystem/LinkedFileUtilities.cs

```diff
@@ -152,5 +152,23 @@ private string GetActiveIntellisenseProjectContextInternal(IVsHierarchy hierarch
                 ? intellisenseProjectName as string
                 : null;
         }
+
+        public static bool TryGetItemIdInSharedHierarchy(IVsHierarchy hierarchy, uint itemId, IVsHierarchy sharedHierarchy, out uint itemIdInSharedHierarchy)
+        {
```

Comment:
ah, will do. Just noticing that the rest of the methods in this helper already follow that pattern.